server/plugin/plugin.go

```diff
@@ -250,6 +275,149 @@ func (p *Plugin) OnDeactivate() error {
+	if !ok || id == 0 {
+		return orgRepo, id, objectType, false
+	}
+
+	objectType, ok = post.GetProp(postPropGithubObjectType).(string)
+	if !ok || objectType == "" {
+		return orgRepo, id, objectType, false
+	}
+
+	return orgRepo, id, objectType, true
+}
+
+func (p *Plugin) ReactionHasBeenAdded(c *plugin.Context, reaction *model.Reaction) {
+	githubEmoji := p.emojiMap[reaction.EmojiName]
+	if githubEmoji == "" {
+		return
+	}
+
+	orgRepo, id, objectType, ok := p.getPostPropsForReaction(reaction)
+	if !ok {
+		return
+	}
+
+	info, appErr := p.getGitHubUserInfo(reaction.UserId)
+	if appErr != nil {
+		if appErr.ID != apiErrorIDNotConnected {
+			p.API.LogError("Error in getting user info", "error", appErr.Message)
+		}
+		return
+	}
+
+	ghClient := p.githubConnectUser(context.Background(), info)
+	switch objectType {
+	case githubObjectTypeIssueComment:
+		if _, _, err := ghClient.Reactions.CreateIssueCommentReaction(context.Background(), orgRepo[0], orgRepo[1], int64(id), githubEmoji); err != nil {
+			p.API.LogDebug("Error found while creating issue comment reaction", err)
+			return
+		}
+	case githubObjectTypeIssue:
+		if _, _, err := ghClient.Reactions.CreateIssueReaction(context.Background(), orgRepo[0], orgRepo[1], int(id), githubEmoji); err != nil {
+			p.API.LogDebug("Error found while creating issue reaction", err)
+			return
+		}
+	case githubObjectTypePRReviewComment:
+		if _, _, err := ghClient.Reactions.CreatePullRequestCommentReaction(context.Background(), orgRepo[0], orgRepo[1], int64(id), githubEmoji); err != nil {
+			p.API.LogDebug("Error found while creating PR review comment reaction", err)
+			return
+		}
+	default:
+		return
+	}
+}
+
+func (p *Plugin) ReactionHasBeenRemoved(c *plugin.Context, reaction *model.Reaction) {
+	githubEmoji := p.emojiMap[reaction.EmojiName]
+	if githubEmoji == "" {
+		return
+	}
+
+	orgRepo, id, objectType, ok := p.getPostPropsForReaction(reaction)
+	if !ok {
+		return
+	}
+
+	info, appErr := p.getGitHubUserInfo(reaction.UserId)
+	if appErr != nil {
+		if appErr.ID != apiErrorIDNotConnected {
+			p.API.LogError("Error in getting user info", "error", appErr.Message)
+		}
+		return
+	}
+
+	ghClient := p.githubConnectUser(context.Background(), info)
+	switch objectType {
+	case githubObjectTypeIssueComment:
+		reactions, _, err := ghClient.Reactions.ListIssueCommentReactions(context.Background(), orgRepo[0], orgRepo[1], int64(id), &github.ListOptions{})
+		if err != nil {
+			p.API.LogDebug("Error getting issue comment reaction list", err)
+			return
+		}
+
+		for _, reactionObj := range reactions {
+			if info.UserID == reaction.UserId && p.emojiMap[reaction.EmojiName] == reactionObj.GetContent() {
+				if _, err = ghClient.Reactions.DeleteIssueCommentReaction(context.Background(), orgRepo[0], orgRepo[1], int64(id), reactionObj.GetID()); err != nil {
+					p.API.LogDebug("Error found while removing issue comment reaction", "error", err.Error())
```

Comment:
```suggestion
					p.API.LogDebug("Error occurred while removing issue comment reaction", "error", err.Error())
```
I think `found` is the wrong word to use. Fix everywhere.